src/applications/personalization/appointments/actions/index.js

```diff
@@ -1,27 +1,212 @@
+import moment from 'moment';
+import { replace, uniq, isEmpty } from 'lodash';
+import environment from 'platform/utilities/environment';
 import { apiRequest } from 'platform/utilities/api';
+import {
+  getCCTimeZone,
+  getVATimeZone,
+} from '~/applications/personalization/dashboard-2/utils/timezone';
+import {
+  FETCH_CONFIRMED_FUTURE_APPOINTMENTS,
+  FETCH_CONFIRMED_FUTURE_APPOINTMENTS_FAILED,
+  FETCH_CONFIRMED_FUTURE_APPOINTMENTS_SUCCEEDED,
+  FUTURE_APPOINTMENTS_HIDDEN_SET,
+  VIDEO_TYPES,
+} from '~/applications/personalization/dashboard-2/constants';
+import MOCK_FACILITIES from '~/applications/personalization/dashboard-2/utils/mocks/appointments/MOCK_FACILITIES.json';
+import MOCK_VA_APPOINTENTS from '~/applications/personalization/dashboard-2/utils/mocks/appointments/MOCK_VA_APPOINTMENTS';
+import MOCK_CC_APPOINTMENTS from '~/applications/personalization/dashboard-2/utils/mocks/appointments/MOCK_CC_APPOINTMENTS';
 
-export const FETCH_APPOINTMENTS = 'FETCH_APPOINTMENTS';
```

Comment:
btw, thanks for removing these since we haven't used them in I don't know how long